src/proposal.ts

```diff
@@ -406,10 +416,28 @@ export class Proposal implements IStateful<IProposalState> {
     )
   }
 
-  public rewards(options: IRewardQueryOptions = {}): Observable < IRewardState[] > {
+  public rewards(options: IRewardQueryOptions = {}): Observable<IRewardState[]> {
     options.proposal = this.id
     return Reward.search(this.context, options)
   }
+
+  public claimRewards(account: Address): Operation<boolean> {
+    const transaction = this.votingMachine().methods.redeem(this.id, account)
```

Comment:
im pretty sure we need to use the Redeemer contract to redeem all the rewards at once, not just the votingMachine redeem

Reply:
:+1:

Reply:
created a seperate issue for that: https://github.com/daostack/client/issues/84